server/db/recreate-schema.sql

```diff
@@ -1,100 +1,154 @@
-);
+--  users definition
 
+-- Drop table
 
+-- DROP TABLE  users;
 
-CREATE TABLE blocks(
-  block_id INT GENERATED ALWAYS AS IDENTITY,
-   block_name VARCHAR(255) NOT NULL,
-   PRIMARY KEY( block_id)
+CREATE TABLE  users (
+	id serial NOT NULL,
+	email varchar(200) NOT NULL,
+	"password" varchar(200) NOT NULL,
+	CONSTRAINT users_pkey PRIMARY KEY (id)
 );
 
+-- Permissions
+
+-- ALTER TABLE  users OWNER TO app_user;
+-- GRANT ALL ON TABLE  users TO app_user;
+
 
+-----------------------------------------------------------------------------
+--  blocks definition
 
+-- Drop table
 
-CREATE TABLE topics(
-   topic_id INT GENERATED ALWAYS AS IDENTITY,
-   topic_name VARCHAR(255) NOT NULL,
-   document_name VARCHAR(255) ,
-   document_link VARCHAR(255) ,
-   PRIMARY KEY(topic_id)
+-- DROP TABLE  blocks;
+
+CREATE TABLE  blocks (
+	block_id int4 NOT NULL GENERATED ALWAYS AS IDENTITY,
+	block_name varchar(255) NOT NULL,
+	CONSTRAINT blocks_pkey PRIMARY KEY (block_id)
 );
-CREATE TABLE block_topics(
-  block_id INT NOT NULL,
-   topic_id INT NOT NULL,
-   CONSTRAINT fk_topics
-      FOREIGN KEY(topic_id)
-	  REFERENCES topics(topic_id),
-    CONSTRAINT fk_blocks
-      FOREIGN KEY(block_id) 
-	  REFERENCES blocks(block_id)
+
+-- Permissions
+
+-- ALTER TABLE  blocks OWNER TO app_user;
+-- GRANT ALL ON TABLE  blocks TO app_user;
+
+
+
+--  volunteers definition
+
+
+-- DROP TABLE  volunteers;
+
+CREATE TABLE  volunteers (
+	volunteer_id int4 NOT NULL GENERATED ALWAYS AS IDENTITY,
+	email varchar(255) NOT NULL,
+	CONSTRAINT volunteers_pkey PRIMARY KEY (volunteer_id)
 );
- 
-
-CREATE TABLE questions(
-   question_id INT GENERATED ALWAYS AS IDENTITY,
-  topic_id  INT NOT NULL,
-   question_text VARCHAR(255) NOT NULL,
-   PRIMARY KEY(question_id),
-   CONSTRAINT fk_topics
-      FOREIGN key(topic_id) 
-	  REFERENCES topics(topic_id)
-   
+
+-- Permissions
+
+-- ALTER TABLE  volunteers OWNER TO app_user;
+-- GRANT ALL ON TABLE  volunteers TO app_user;
+
+
+--  topics definition
+
+
+-- DROP TABLE  topics;
+
+CREATE TABLE  topics (
+	topic_id int4 NOT NULL GENERATED ALWAYS AS IDENTITY,
+	topic_name varchar(255) NOT NULL,
+	document_name varchar(255) NOT NULL,
+	document_link varchar(255) NOT NULL,
+	block_id int4 NULL,
+	CONSTRAINT topics_pkey PRIMARY KEY (topic_id),
+	CONSTRAINT topics_fk FOREIGN KEY (block_id) REFERENCES blocks(block_id)
 );
-CREATE TABLE volunteers(
-   volunteer_id INT GENERATED ALWAYS AS IDENTITY,
-   email VARCHAR(255) NOT NULL,
-   PRIMARY KEY(volunteer_id)
-   
+
+-- Permissions
+
+-- ALTER TABLE  topics OWNER TO app_user;
+-- GRANT ALL ON TABLE  topics TO app_user;
+
+
+--  "volunteer_Blocks" definition
+
+-- DROP TABLE  "volunteer_Blocks";
+
+CREATE TABLE  volunteer_blocks (
+	volunteer_block_id int4 NOT NULL GENERATED ALWAYS AS IDENTITY,
+	volunteer_id int4 NOT NULL,
+	assessment_start timestamp NULL,
+	assessment_end timestamp NULL,
+	block_id int4 NULL,
+	CONSTRAINT volunteer_topic_pkey PRIMARY KEY (volunteer_block_id),
```

Comment:
Because you have renamed this table the volunteer_topic table still exists and therefore the volunteer_topic_pkey still exists so you cannot run this script over the top of the previous one. Either rename this key or drop the volunteer_topic table.